config/initializers/new_framework_defaults.rb

```diff
@@ -17,6 +15,3 @@
 # Make Ruby 2.4 preserve the timezone of the receiver when calling `to_time`.
 # Previous versions had false.
 ActiveSupport.to_time_preserves_timezone = false
-
-# Do not halt callback chains when a callback returns false. Previous versions had true.
-ActiveSupport.halt_callback_chains_on_return_false = true
```

Comment:
was this change and the one above necessary? both of the settings that were here seem reasonable?

Reply:
@akegan both of those options were removed from Rails. The first one actually didnt have any effect since Rails 5.1. The second one was replaced by explicitly calling `throw :abort` in Rails controller.

Reply:
thanks!